Test Math.cpp

Carrotstrip opened the pull request:
I wouldn't call this entirely thorough, but I wanted to get it into your hands to make sure it's the kind of thing you're looking for. I noticed that you do random generation with a big for loop in QuaternionTest; is that something we'd like to see in MathTest as well?

hhirsch commented:
@Carrotstrip Can you add this to the test suite so it will be run? This way the reviewer does not have to pull your branch to see the test results.

You can call the file Math.cpp. Our convention is to have names in camel case and since the directory already signifies that it is a test you can drop the test prefix.

Carrotstrip commented:
Changed the file name to MathTest.cpp (following the pattern of QuaternionTest.cpp, let me know if you'd rather it just be Math.cpp) and added the test to CMakeLists.txt.

hhirsch commented:
@Carrotstrip Thanks a lot.
